app/helpers/application_helper.rb

```diff
@@ -51,8 +51,8 @@ def load_select_search_type
 
   def load_select_search_status
     if action_name.eql?("show") || controller_name.eql?("favorites")
-      Post.statuses.keys.map do |key|
-        [t(key.to_s), key]
+      Post.statuses.map do |key, value|
```

Comment:
ngắn ntn thì viết trên 1 dòng đi e
```suggestion
      Post.statuses.map{|key, value| [t(key.to_s), value]
```

Reply:
vang a